internal/verifier/verifier_test.go

```diff
@@ -115,12 +107,10 @@ func TestVerifyHasNoValues(t *testing.T) {
 	}
 
 	// We don't expect any verification to happen.
-	result, err := verifier.Verify()
+	err := verifier.Verify()
```

Comment:
Can this be changed into `if err := ...; err != nil {...}`?

Reply:
Done.